index.js

```diff
@@ -184,4 +189,42 @@ const addOrRemoveParticipant = (message, chatId, usernames, ref, res) => {
     console.log(err);
     return res.end('Error: ', err);
   });
-};
\ No newline at end of file
+};
+
+const setTeam = (message, chatId, ref, res) => {
+  const { text } = message;
+  const emptyIndex = text.indexOf(' ');
+  const teamName = text.subtr(0, emptyIndex);
+  const teamUsers = text.subtr(emptyIndex);
```

Comment:
Won't the text be something like `/setTeam teamName @user @user2`? If the text would be like this then this logic won't work since the first space character would be the one between the command and the team name.